src/navigation.tpl

```diff
@@ -54,6 +54,7 @@
                   </li>
                   <li class="section"><a class="header" href="howto.html">How to...</a>
                     <ul class="sub">
+                      <li><a href="howto-charm-with-docker.html">Charm with Docker</a></li>
```

Comment:
This strikes me as a user centric how-to section. We should probably make a new How-To section targeted at developers w/ our developer how-to's.